api/src/backend/tasks/tasks.py

```diff
@@ -75,6 +77,7 @@ def _perform_scan_complete_tasks(tenant_id: str, scan_id: str, provider_id: str)
     )
     chain(
         perform_scan_summary_task.si(tenant_id=tenant_id, scan_id=scan_id),
+        aggregate_daily_severity_task.si(tenant_id=tenant_id, scan_id=scan_id),
```

Comment:
I have doubts about this. This for sure needs to be executed AFTER scan summary, but does it really need to block the outputs one?